prow/config.yaml

```diff
@@ -526,6 +526,8 @@ deck:
   spyglass:
     size_limit: 500000000 # 500MB
     gcs_browser_prefix: https://gcsweb.build.kyma-project.io/gcs/
+    testgrid_config: gs://k8s-testgrid/config
```

Comment:
As far as I know, after recent changes, it must be added to https://github.com/kyma-project/test-infra/blob/master/templates/templates/prow-config.yaml

...and yes, I think it makes everything more complicated...

Reply:
Good find, I'll change that.